src/any_llm/utils/structured_output.py

```
@@ -143,30 +143,42 @@ def parse_responses_output(response: Any, response_format: type) -> ParsedRespon
+
+    ``output_format`` is either a structured-output type (Pydantic ``BaseModel`` subclass or
+    dataclass), in which case ``parsed_output`` holds the typed object, or a raw Anthropic
+    ``output_config`` dict, in which case ``parsed_output`` holds the parsed JSON (a plain
+    ``dict``/``list``) and the generics are parametrized as ``Any``.
     """
+    import json
+
     from anthropic.types import TextBlock
     from anthropic.types.parsed_message import ParsedMessage, ParsedTextBlock
 
+    def _parse(text: str) -> Any:
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_parse`

(ANN401)